gourmet/gtk_extras/TextBufferMarkup.py

```diff
@@ -253,7 +264,7 @@ def setup_default_tags (self):
     def get_selection (self):
         bounds = self.get_selection_bounds()
         if not bounds:
-            iter=self.get_iter_at_mark(self.insert)
+            iter=self.get_iter_at_mark(self.insert_)
```

Comment:
That one drove me crazy `self.insert_` is a `Pango.TextMark`; `self.insert` is a function inherited from `Gtk.TextBuffer`, used to insert the markups around the text.

So if you have:
```html
blablabla
```
it will insert the markup around the (start, end) provided by each attribute in `self.attrList`:
```html
bla<strong>bla</strong>bla
```